fastlane/lib/fastlane/swift_fastlane_api_generator.rb

```diff
@@ -13,28 +13,18 @@ def initialize(command_line_tool_name: nil, swift_class: nil, swift_protocol: ni
     end
   end
 
-  class SwiftFastlaneAPIGenerator
-    DEFAULT_API_VERSION_STRING = "0.9.1"
-    attr_accessor :tools_option_files
-    attr_accessor :actions_not_supported
-    attr_accessor :action_options_to_ignore
-    attr_accessor :target_output_path
-    attr_accessor :generated_paths # stores all file names of generated files (as they are generated)
+  class SwiftAPIGenerator
```

Comment:
Hmm, should `SwiftAPIGenerator` class be here as an empty one? 🤔There's the same one but with the implementation below (line 115).

Reply:
Ahhhh, I needed to declare it before it was getting used so I stubbed it up at the top and then reopened it up below to make the changes 😛 I should probably make this prettier